phpBB/phpbb/notification/method/webpush.php

```diff
@@ -140,13 +140,7 @@ public function notify()
 		{
 			$data = $notification->get_insert_array();
 			$data += [
-				'push_data'		=> json_encode([
-					'heading'	=> $this->config['sitename'],
-					'title'		=> strip_tags($notification->get_title()),
-					'text'		=> strip_tags($notification->get_reference()),
-					'url'		=> htmlspecialchars_decode($notification->get_url()),
-					'avatar'	=> $notification->get_avatar(),
-				]),
+				'push_data'				=> json_encode(array_merge($notification->get_insert_array(), ['notification_type_name' => $notification->get_type()])),
```

Comment:
I'm confused by this. Where is all this push data going now? Looks like its gone? Is that supposed to be `$notification->get_insert_array()` again? This looks like it's an Ouroboros, lol. It looks sort of like it's:
```php
$data = $notification->get_insert_array();
$data += ['push_data' => $data]
```
like it's putting itself in iitself.

Reply:
Ok maybe I'm starting to get this, lol. But if using using `$notification->get_insert_array()` is intended twice here, maybe just use `$data` the second time?

```php
'push_data' => json_encode(array_merge($data, ['notification_type_name' => $notification->get_type()])),
```

Reply:
It might actually make sense to add a migration and adjust the table instead.

Reply:
I've cleaned this up a bit and it should be less confusing now